pynucastro/screening/screen.py

```diff
@@ -586,3 +609,32 @@ def potekhin_1998(state, scn_fac):
     scor = np.exp(h12)
 
     return scor
+
+
+def screening_check(check_func=debye_huckel, threshold: float = 1.01):
+    """A decorator factory that wraps a screening function with a
+    check that determines whether that function can be skipped for a
+    given plasma state and screening pair.
+
+    :param func: the function to check against the threshold
+    :param threshold: the threshold to check against. If screen_check
+    is less than the threshold, skip screen_func
```

Comment:
Sphinx wants this to be indented
```suggestion
    :param threshold: the threshold to check against. If screen_check
                      is less than the threshold, skip screen_func
```